src/messages.rs

```diff
@@ -458,9 +530,9 @@ impl Debug for HopMessage {
 impl Debug for SignedMessage {
     fn fmt(&self, formatter: &mut Formatter) -> fmt::Result {
         write!(formatter,
-               "SignedMessage {{ content: {:?}, public_id: {:?}, signature: .. }}",
+               "SignedMessage {{ content: {:?}, {} signatures }}",
                self.content,
-               self.public_id)
+               self.signatures.len())
```

Comment:
Maybe consider adding the list of names that signed the message, could be useful in debugging.